accelerator/.config/ALZ-Powershell-Auto.config.json

```diff
@@ -560,6 +636,105 @@
             }
           ]
         },
+        "parAzBastionName": {
```

Comment:
I understand the reason for adding these. I'm just wondering if we are potentially impacting existing users upgrading a single region deployment?